app/build.gradle

```
@@ -34,8 +34,7 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.6.0'
```

Deberíamos usar la última versión disponible de Retrofit.
```suggestion
    implementation 'com.squareup.retrofit2:retrofit:2.6.2'
```